### ISSUE TITLE
Luma theme shipping tool tip icon cut from leftside

### ISSUE THREAD

**Issue by abrarpathan19** [magento/magento2#20838]
### Preconditions (*)

1. Magento 2.3 and Magento 2.2.X

### Steps to reproduce (*)

1. Go to checkout- shipping page 
2. Find the field shippingAddress telephone tool tip icon 

### Expected result (*)

1. http://prntscr.com/meme4u

### Actual result (*)

1. http://prntscr.com/memgdr

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@abrarpathan19 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by suryakant-krish** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @suryakant-krish. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @suryakant-krish. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
Hi @nainesh2jcommerce. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
@suryakant-krish Thank you for verifying the issue.

Unfortunately, not enough information was provided to acknowledge ticket. Please consider adding the following:
- [ ] Add `"Reproduced on "` label(s) to this ticket based on verification result

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by suryakant-krish** [comment]
@magento-engcom-team,

I have reviewed in my local instance but I need to verify in Magento instance please provide me instance.

**Comment by nainesh2jcommerce** [comment]
reproduce on local
![image](https://user-images.githubusercontent.com/18118539/52041467-8b8ef200-2560-11e9-8bb4-a8109493af0a.png)

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @nainesh2jcommerce
Thank you for verifying the issue! :+1: Your confirmation will help us to acknowledge and process this report.

**Comment by suryakant-krish** [comment]
Look in this image.

![confirm-issue](https://user-images.githubusercontent.com/43638601/52041637-02c48600-2561-11e9-9add-80ee9d3e516a.jpg)

**Comment by shikhamis11** [comment]
@suryakant-krish I am not able find difference between expected and old one.
This seems not an issue

**Comment by shikhamis11** [comment]
@abrarpathan19 Please wait for the issue confirmation before creating pull requests as this may lead to additional work for maintainers as well as contributors.

**Comment by suryakant-krish** [comment]
@shikhamis11 

Could you please save this image in your local machine and zoom 200%  because it's SVG fonts used so it is high quality. 

Image Link : https://prnt.sc/memgdr

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by suryakant-krish** [comment]
@magento-engcom-team,

I have confirmed the first issue and added labels and do testing on a local machine and provided a screenshot for that. but how you confirm issue @nainesh2jcommerce.

Could you please refer all conversion from top to bottom.

**Comment by ghost** [comment]
@suryakant-krish what's wrong here ?
![selection_304](https://user-images.githubusercontent.com/40756439/52045244-c7b85780-254c-11e9-9b7f-583d52a8a8a6.png)

**Comment by abrarpathan19** [comment]
@engcom-backlog-nazar This is the screenshot when browser zoom size will be 100%,  If you do ctnl + zoom in it you can’t reproduce the issue.
Please find the attachment 

![11111111111](https://user-images.githubusercontent.com/43603387/52046674-4e7d2c80-256d-11e9-8fd0-160d91705f30.jpg)


Also @nainesh2jcommerce and @suryakant-krish has already reproduce this issue. can you please review their screenshots too.

**Comment by ghost** [comment]
@abrarpathan19 but why i see another result ? and please for simple issue describe problem clearly
![selection_305](https://user-images.githubusercontent.com/40756439/52049065-81b3c180-2555-11e9-9d4e-014d1f198219.png)

**Comment by suryakant-krish** [comment]
@engcom-backlog-nazar,

I reproduced in windows system chrome browser latest version with the 1920 resolution.

Give me an instance for that I will re-confirm that issue.

**Comment by ghost** [comment]
@suryakant-krish ok now clear, this issue related to resolution.

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @shikhamis11 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98004` were created

**Issue Available**: @shikhamis11, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19. Thank you for your report. 
The issue has been fixed in magento/magento2#20839 by @abrarpathan19 in 2.3-develop branch
Related commit(s):
 - [b76711cfc35e2bd07b734726ad244a7054453375](https://github.com/magento/magento2/commit/b76711cfc35e2bd07b734726ad244a7054453375)

The fix will be available with the upcoming 2.3.2 release.

**Comment by dmytro-ch** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [Magento instance](https://i-20838-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-20838-2-2-develop.instances.magento-community.engineering/admin](https://i-20838-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.
